src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -198,7 +206,7 @@ def get_tags(
     return fields
 
 
-def get_context(group: Group) -> str:
+def get_context(group: Group, rules: list[Rule] | None = None) -> str:
```

Comment:
this logic is messy, but we remove `Users Affected` if possible because it requires a Snuba query